packages/contact-center/store/src/store.ts

```diff
@@ -1,67 +1,63 @@
 import {makeAutoObservable, observable} from 'mobx';
 import Webex from 'webex';
-import {
-  IContactCenter,
-  Profile,
-  Team,
-  WithWebex,
-  IdleCode,
-  InitParams,
-  IStore
-} from './store.types';
+import {IContactCenter, Profile, Team, WithWebex, IdleCode, InitParams, IStore} from './store.types';
 
 class Store implements IStore {
   teams: Team[] = [];
   loginOptions: string[] = [];
   cc: IContactCenter;
   idleCodes: IdleCode[] = [];
   agentId: string = '';
+  selectedLoginOption: string = '';
 
   constructor() {
     makeAutoObservable(this, {cc: observable.ref});
   }
 
   registerCC(webex: WithWebex['webex']): Promise<void> {
     this.cc = webex.cc;
-    return this.cc.register().then((response: Profile) => {
-      this.teams = response.teams;
-      this.loginOptions = response.loginVoiceOptions;
-      this.idleCodes = response.idleCodes;
-      this.agentId = response.agentId;
-    }).catch((error) => {
-      console.error('Error registering contact center', error);
-      return Promise.reject(error);
-    });
+    return this.cc
+      .register()
+      .then((response: Profile) => {
+        this.teams = response.teams;
+        this.loginOptions = response.loginVoiceOptions;
+        this.idleCodes = response.idleCodes;
+        this.agentId = response.agentId;
+      })
+      .catch((error) => {
+        console.error('Error registering contact center', error);
+        return Promise.reject(error);
+      });
   }
 
   init(options: InitParams): Promise<void> {
-    if('webex' in options) {
+    if ('webex' in options) {
```

Comment:
question. Is `webex` supposed to be an existing SDK instance?

Reply:
This is not a change introduced in this PR. Context on this change - developers can pass either access_token or webex object to the widgets while initialising.

Yes this is an existing sdk instance that developers can pass. This will help them to bring in the widgets to an existing application that already uses sdk.